ocs_ci/helpers/dr_helpers.py

```diff
@@ -73,3 +73,97 @@ def relocate(preferred_cluster, drpc_name, namespace):
     drpc_obj.wait_for_phase(constants.STATUS_RELOCATED)
 
     config.switch_ctx(prev_index)
+
+
+def get_drpc_name(namespace):
+    """
+    Get the DRPC Resource Name
+    Args:
+        namespace (str): Name of namespace
+    Returns:
+        str: DRPC resource name
+    """
+
+    drpc_obj = ocp.OCP(kind=constants.DRPC, namespace=namespace,).get()[
+        "items"
+    ][0]
+    return drpc_obj["metadata"]["name"]
+
+
+def get_drpolicy_name():
+    """
+    Get DRPolicy Name
+    Returns:
+        str: DRPolicy name
+    """
+
+    drpolicy_obj = ocp.OCP(kind=constants.DRPOLICY).get()["items"][0]
```

Comment:
What if there are multiple DR policy in the setup?
I suggest to fetch DR policy based on the DRPC resource.